apps/meteor/server/services/team/service.ts

```diff
@@ -473,7 +473,8 @@ export class TeamService extends ServiceClassInternal implements ITeamService {
 		await Rooms.setTeamDefaultById(rid, isDefault);
 
 		if (room.teamDefault) {
-			const teamMembers = await this.members(uid, room.teamId, true, undefined, undefined);
+			const MAX_NUMBER_OF_AUTO_JOIN_MEMBERS = 1000;
```

Comment:
Even though this is the only constant being used in this file, I'd suggest adding this to a `constants.ts` file. You can create one in `apps/meteor/server/services/team/lib/constants.ts`

Reply:
great idea!